caktus_primer/templates/styleguide/styleguide-navigation_patterns-search.html

```diff
@@ -0,0 +1,50 @@
+{% extends "styleguide/styleguide.html" %}
+
+{% load styleguide_tags %}
+
+{% block styleguide_content %}
+<h1 class="styleguide-page-title">Search Components</h1>
+
+{% example "Simple Search" %}
+    <!-- TEMPLATE -->
+    <form>
+        <fieldset>
+            <div class="search">
+                {{ everything_bagel_form.search }}
+                <div class="search-submit">
+                    <input type="submit" value="" formnovalidate>
+                    <svg id="Magnify" data-name="Magnify" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 55.69 55.69" ><defs><style>.cls-1{fill:transparent;stroke-width:3px;}.cls-1,.cls-2,.cls-3{stroke-miterlimit:10;}.cls-2,.cls-3{fill:none;}.cls-2{stroke-width:2px;}.cls-3{stroke-width:4px;}</style></defs><title>Magnify</title><circle class="cls-1" cx="19.09" cy="19.09" r="17.59"/><line class="cls-2" x1="31.69" y1="31.69" x2="54.28" y2="54.28"/><line class="cls-3" x1="36.69" y1="36.69" x2="54.28" y2="54.28"/></svg>
+
+                </div>
+            </div>
+        </fieldset>
+    </form>
+{% endexample %}
+
+{% example "Category Search" %}
+    <!-- TEMPLATE -->
+    <form>
+        <fieldset>
+            <div class="category-search">
+                <div class="search-select">
+                    <select name="category" onchange="this.parentNode.lastElementChild.innerHTML = this.options[this.selectedIndex].innerHTML">
```

Comment:
This JS should be in our new JS module for primer, it would be the first code to go in there.